build.gradle

```diff
@@ -87,8 +87,23 @@ subprojects {
           }
         }
       }
+      repositories {
+        maven {
+          name = 'localTest'
+          url = "$rootProject.buildDir/local-test-repo"
```

Comment:
Please use `${rootProject.buildDir}` as it is more clear where the substitution ends.

Reply:
This still seems outstanding?